src/view/ViewCommandHandlers.js

```diff
@@ -22,7 +22,7 @@
  */
 
 /*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, maxerr: 50 */
-/*global define, $ */
+/*global define, brackets: true, $ */
```

Comment:
`brackets` is not writable here, so it should be `brackets: false`. additionally, it would probably be better if either it was just `brackets` or other globals should be explicitly marked as read-only as well.